packages/stryker-jest-runner/src/JestTestRunner.ts

```diff
@@ -15,19 +16,31 @@ export default class JestTestRunner implements TestRunner {
     // Get jest configuration from stryker options and assign it to jestConfig
     this.jestConfig = options.strykerOptions.jest.config;
 
+    // Get enableFindRelatedTests from stryker jest options or default to true
+    this.enableFindRelatedTests = options.strykerOptions.jest.enableFindRelatedTests;
+    if (this.enableFindRelatedTests === undefined) {
+      this.enableFindRelatedTests = true;
+    }
+
+    if (this.enableFindRelatedTests) {
+      this.log.debug('Running jest with --findRelatedTests flag. Set jest.enableFindRelatedTests to false to run all tests on every mutant.');
+    } else {
+      this.log.debug('Running jest without --findRelatedTests flag. Set jest.enableFindRelatedTests to true to run only relevant tests on every mutant.');
+    }
+
     // basePath will be used in future releases of Stryker as a way to define the project root
     // Default to process.cwd when basePath is not set for now, should be removed when issue is solved
     // https://github.com/stryker-mutator/stryker/issues/650
     this.jestConfig.rootDir = options.strykerOptions.basePath || process.cwd();
     this.log.debug(`Project root is ${this.jestConfig.rootDir}`);
   }
 
-  public async run(): Promise<RunResult> {
+  public async run(options?: RunOptions): Promise<RunResult> {
```

Comment:
I think I initially did that because tests started failing and I was unsure whether or not running without `RunOptions` was a case. I've taken a look at the other `*TestRunnerSpec`s and I've added some `RunOptions` to the Jest ones. Is this correct?